.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,6 +6,11 @@
 - [ ] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
 - [ ] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
 - [ ] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) beforehand.
-- [ ] Run the shell script `./bin/generate-samples.sh`to update all Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. These must match the expectations made by your contribution. You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
+- [ ] Run run 
+  ```
+  mvn clean package
+  ./bin/generate-samples.sh
```

Comment:
`docs/generators/*` also checked against head so it make sense to add `./bin/utils/export_docs_generators.sh` with note that this is only required when input parameters for generator are changed

Reply:
good idea, I will add it. an experienced user will be able to save time by skipping this if he knows the docs dont change.